.env.example

```diff
@@ -2,4 +2,4 @@ export API_KEY_INFURA="YOUR_API_KEY_INFURA"
 export EOA="YOUR_EOA_ADDRESS"
```

Comment:
`export API_KEY_INFURA="YOUR_API_KEY_INFURA"` is no longer required.

Reply:
correct